### PR TITLE
Fix/clear on close type

### PR DESCRIPTION
**Pull request by capyq**
Fix typo in onClose prop coming from common-ui.

Related PR (blocking): https://github.com/gridsuite/commons-ui/pull/641

### CODE REVIEW

**Comment by TheMaskedTurtle** [comment]
Confict to manage with Mathieu

File: src/components/menus/directory-tree-contextual-menu.tsx [hunk 1]
```diff
@@ -96,7 +96,7 @@ export default function DirectoryTreeContextualMenu(props: Readonly<DirectoryTre
 
     const [renameCB, renameState] = useDeferredFetch(
         renameElement,
-        () => handleCloseDialog(null, null),
+        () => handleCloseDialog(),
```
Comment (TheMaskedTurtle) [inline src/components/menus/directory-tree-contextual-menu.tsx]: Why not just pass handleCloseDialog, it's already a callback

File: src/components/menus/directory-tree-contextual-menu.tsx [hunk 2]
```diff
@@ -384,7 +384,7 @@ export default function DirectoryTreeContextualMenu(props: Readonly<DirectoryTre
                 return (
                     <FilterCreationDialog
                         open
-                        onClose={handleCloseDialog}
+                        onClose={() => handleCloseDialog()}
```
Comment (TheMaskedTurtle) [inline src/components/menus/directory-tree-contextual-menu.tsx]: Why not just pass handleCloseDialog, it's already a callback

File: src/components/menus/directory-tree-contextual-menu.tsx [hunk 3]
```diff
@@ -411,7 +411,12 @@ export default function DirectoryTreeContextualMenu(props: Readonly<DirectoryTre
     return (
         <>
             {open && (
-                <CommonContextualMenu {...others} menuItems={buildMenu()} open={open && !hideMenu} onClose={onClose} />
+                <CommonContextualMenu
+                    {...others}
+                    menuItems={buildMenu()}
+                    open={open && !hideMenu}
+                    onClose={() => onClose()}
```
Comment (TheMaskedTurtle) [inline src/components/menus/directory-tree-contextual-menu.tsx:413]: Why not just pass onClose, it's already a callback

Comment (thangqp) [reply src/components/menus/directory-tree-contextual-menu.tsx:413]: TS2322: Type
(nextSelectedDirectoryId?: string | null | undefined) => void
is not assignable to type
(event: {}, reason: "backdropClick" | "escapeKeyDown") => void
Types of parameters nextSelectedDirectoryId and event are incompatible.
Type {} is not assignable to type string
Menu. d. ts(52, 3): The expected type comes from property onClose which is declared here on type
IntrinsicAttributes & Readonly<CommonContextualMenuProps>

This is a proposition of Quentin as a hack for type compatibility

File: src/components/menus/directory-tree-contextual-menu.tsx (hunk 1 above)
Comment (thangqp) [reply src/components/menus/directory-tree-contextual-menu.tsx]: DONE in https://github.com/gridsuite/gridexplore-app/pull/540/commits/6c117350141ea6709558c2e41df7797be49220b2

File: src/components/menus/directory-tree-contextual-menu.tsx (hunk 2 above)
Comment (thangqp) [reply src/components/menus/directory-tree-contextual-menu.tsx]: DONE in https://github.com/gridsuite/gridexplore-app/pull/540/commits/6c117350141ea6709558c2e41df7797be49220b2

**Review by TheMaskedTurtle — APPROVED** [review]

**Pull request merged**